docs/nx-cloud/private/get-started.md

```diff
@@ -1,23 +1,62 @@
-# Running Nx Cloud on Prem
+# Running Nx Cloud Enterprise
 
-Nx Cloud can be deployed to your cloud, which gives you full control of your data, with no external API calls.
+We offer multiple ways of running NxCloud for our Enterprise customers. The below options are listed in the order we recommend them, from easiest to most complex in
+terms of set-up and maintenance for your team. Please carefully consider your organization's requirements and level of infrastructure expertise before deciding on
+a deployment option.
 
-This solution is included as part of the Nx Enterprise package. You can learn more about it at [nx.app/enterprise](https://nx.app/enterprise).
+## Clusters managed by us
 
-Nx Cloud can be deployed in two ways:
+### Multi-tenant
 
-- Using Kubernetes (several containers working together)
-- Using a single standalone container (NOT RECOMMENDED)
+The quickest way and easiest way to get up and running with NxCloud is using one of our existing secure, multi-tenant managed clusters:
 
-The flags and the capabilities are the same between the two, but the Kubernetes setup is more robust and better
-documented.
+- https://nx.app/
+- Or, if you'd like all of your data to be hosted in Europe, you can use https://eu.nx.app/
+
+You get the same level of security, dedicated support, SSO/SAML auth options and predictable seat-based pricing as all our other hosting options, but you won't have
+to manage the instance yourself.
+
+We also offer an uptime SLA guarantee of 99.98% for our Enterprise customers, SOC certificates on request, and we're happy to meet with your security teams if they
+have questions, or fill in security questionnaires. We also maintain a [Status Page here](https://status.nx.app/).
+
+To start with this option, it's as easy as running `npx nx connect` in your Nx workspace!
+
+### Single-tenant instance
+
+If you have very specific requirements, then we can also offer to host NxCloud for you in an isolated/single-tenant cluster.
+
+We'll be able to discuss specific requirements such as:
+
+- Specific regions you want your data to be in
+- Network isolation / dedicated VPCs
+- Dedicated instances
+- Storage encryption
+- Storage replication / redundancy
+- No external API calls to any services outside of the cluster we set-up for you
+
+This would be a "best of both worlds" option, as it would free you up from managing the instance yourself, but you will get to define specific parameters of how it should it run.
+Your data and the NxCloud will run in complete isolation and will only serve your company. Once you let us know you'd like this option, depending on the agreed requirements,
+it might take a few days to get it set up.
+
+If you'd like to use any of the above options, please mention this to your Enterprise support contact, and they'll be able to set you up with an unlimited use license.
```

Comment:
Maybe link to the Enterprise contact form?

Reply:
```suggestion
If you'd like to use any of the above options, please mention this to your [Enterprise support contact](https://nx.app/enterprise?utm_source=nx.dev), and they'll be able to set you up with an unlimited use license.
```

Reply:
I removed this line, but you're right, I'll make sure to include a link to the form at the top